Student/delivery/http/student_http_test.go

```diff
@@ -19,7 +19,14 @@ import (
 	"testing"
 )
 
-func TestStudentHandler_GetByID(t *testing.T) {
+
+const failureMessage = "failed to read from message"
+const domaindotstudent = "*domain.Student"
```

Comment:
camel case naming convention and naming suggestions:
domaindotstudent -> studentType
apistudents -> putStudentPath
sapistudent -> postStudentPath
sapistudents -> getStudentPath